src/LogicApp/LogicApp/help/Remove-AzIntegrationAccountAssembly.md

```diff
@@ -0,0 +1,152 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.LogicApp.dll-Help.xml
+Module Name: Az.LogicApp
+online version: https://docs.microsoft.com/en-us/powershell/module/az.logicapp/remove-azintegrationaccountassembly
+schema: 2.0.0
+---
+
+# Remove-AzIntegrationAccountAssembly
+
+## SYNOPSIS
+Removes an integration account assembly.
+
+## SYNTAX
+
+### ByIntegrationAccount (Default)
+```
+Remove-AzIntegrationAccountAssembly -ResourceGroupName <String> -ParentName <String> -Name <String>
```

Comment:
Missing passthru? Please rerun help generation for the two removes as it looks like you're missing a parameter in this file.

Reply:
Regenerated